spec/models/refinery/jobs/job_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper'
+
+module Refinery
+  module Jobs
+    describe Job, type: :model do
+      let(:job) {FactoryGirl.create(:job)}
+      describe 'validations' do
+        it 'requires title and description' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.